esp/services/ws_fs/ws_fsBinding.cpp

```diff
@@ -393,12 +393,15 @@ int CFileSpraySoapBindingEx::downloadFile(IEspContext &context, CHttpRequest* re
         if (!context.validateFeatureAccess(FILE_SPRAY_URL, SecAccess_Full, false))
             throw MakeStringException(ECLWATCH_FILE_SPRAY_ACCESS_DENIED, "Failed to download file. Permission denied.");
 
-        StringBuffer netAddressStr, osStr, pathStr, nameStr;
+        StringBuffer netAddressStr, osStr, pathStr, nameStr, dropZoneName;
         request->getParameter("NetAddress", netAddressStr);
         request->getParameter("OS", osStr);
         request->getParameter("Path", pathStr);
         request->getParameter("Name", nameStr);
+        request->getParameter("DropZoneName", dropZoneName);
 
+        if (getDropZoneScopePermissions(context, dropZoneName, netAddressStr, pathStr) < SecAccess_Read)
```

Comment:
see comments elsewhere. Only 1 of either dropZoneName or netAddressStr is valid depending on whehter containerized or not, it would be clearer for the impls. that use 1 to have 1 param, and for the caller to check and use the correct one.

Reply:
Is this related to: https://github.com/hpcc-systems/HPCC-Platform/pull/16710#discussion_r1029416424 ?

Reply:
Yes